packages/ffe-tabs-react/src/Tab.md

```diff
@@ -1,52 +1,40 @@
-# Tabs
-
-Tabs er en komponent for å lage knapper med fane-funksjonalitet - gruppering av innhold der én gruppe er synlig og de øvrige er skjult.
+Tab er en komponent for å lage knapper med fane-funksjonalitet - gruppering av innhold der én gruppe er synlig og de øvrige er skjult.
 
 ```js
-<Tab>
-    Dette er en tab button
-</Tab>
+<Tab>Dette er en fane</Tab>
 ```
 
-Aktiv/valgt tab kan vises ved å sende inn `selected`:
+Aktiv/valgt fane kan vises ved å sende inn `selected`:
 
 ```js
-<Tab selected={true}>
-    Dette er en valgt tab button
-</Tab>
+<Tab selected={true}>Dette er en valgt fane</Tab>
 ```
 
-`ghost` eller `condensed` kan sendes inn for å rendre henholdsvis en transparent eller kompakt tab:
+`condensed` kan sendes inn for å rendre en kompakt fane:
 
 ```js
 const { Grid, GridRow, GridCol } = require('../../ffe-grid-react/lib');
 
 <Grid>
-    <GridRow
-        background='sand'
-        topPadding={true}
-    >
+    <GridRow>
         <GridCol sm={12}>
             <div>
-                <Tab
-                    ghost={true}
-                    condensed={true}
-                >
-                    Dette er en transparent og kompakt tab button
-                </Tab>
+                <Tab condensed={true}>Dette er en kompakt fane</Tab>
             </div>
         </GridCol>
     </GridRow>
-</Grid>
+</Grid>;
 ```
 
 I tillegg kan man sende inn egendefinerte props for å legge til clickhandlere andre attributter:
```

Comment:
På flisespikkerisiden så mangler det et 'og' i denne setningen :)

Reply:
Dårlig gjort å gi deg skylden for en setning som sto der fra før. Setter den til approved :)

Reply:
Godt observert! Jeg fikser det. :)